stories/Welcome-draft.md

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { linkTo } from '@storybook/addon-links';
+
+TODO: Display this as a proper MDX file.
+
+# Welcome to the MarkLogic Design System Storybook
```

Comment:
It might make sense to have a brief overview of what this storybook contains and what it is for.